includes/payment-methods/class-wc-stripe-upe-payment-method-giropay.php

```diff
@@ -19,8 +19,9 @@ class WC_Stripe_UPE_Payment_Method_Giropay extends WC_Stripe_UPE_Payment_Method
 	 */
 	public function __construct( $token_service ) {
 		parent::__construct( $token_service );
-		$this->stripe_id   = self::STRIPE_ID;
-		$this->title       = 'Pay with Giropay';
-		$this->is_reusable = true; // TODO: is it?
+		$this->stripe_id            = self::STRIPE_ID;
+		$this->title                = 'Pay with Giropay';
+		$this->is_reusable          = true; // TODO: is it?
```

Comment:
```suggestion
		$this->is_reusable          = false;
```
This should also be [false](https://stripe.com/payments/payment-methods-guide#giropay).